src/blocks/PressCardBlock/index.tsx

```diff
@@ -0,0 +1,71 @@
+// @ts-ignore
```

Comment:
): can we write some types for this, or in _some_ other way avoid it than turning off the compiler?